### PR TITLE
Live db update

### PR DESCRIPTION
**Pull request by Shakir-Sopra**
Hi Warren

I have updated delete statements keeping old delete statements.

Please ignore the old code which by mistake I deleted from my branch locally deleted by mistake old code so wanted to keep old update statement in the repository.

Thanks for your help on this.
Regards
Shakir

### CODE REVIEW

File: Live_data_update.sql [hunk 1]
```diff
@@ -19,3 +21,85 @@ update cqc."Establishment" set "IsRegulated"=True,"LocationID"='1-5400068426' wh
...
+-- https://trello.com/c/Ypwr3hZ4/55-remove-parent-cohort-2-users-from-asc-wds
+
+--Deletion data view before we delete the Establishment and related records
+
+-- Selection based on Tribal ID [196840,170258]
+
+
+select "TribalID"   from cqc."Establishment" where "NmdsID" IN ('E302906','G260465');
+
+SELECT * FROM cqc."Worker" where "TribalID" in (select "TribalID" from cqc."Establishment" where "NmdsID" IN ('E302906','G260465')) ;
```
Comment (wozzer72) [inline Live_data_update.sql]: Why are you returning the whole of the records? Isn't it enough to return the count of records?

File: Live_data_update.sql [hunk 2]
```diff
@@ -19,3 +21,85 @@ update cqc."Establishment" set "IsRegulated"=True,"LocationID"='1-5400068426' wh
...
+--- Deletion Verification 
+
+-- Selection based on Tribal ID [196840,170258]
+
+
+select "TribalID"   from cqc."Establishment" where "NmdsID" IN ('E302906','G260465');
+
+SELECT * FROM cqc."Worker" where "TribalID" in (select "TribalID" from cqc."Establishment" where "NmdsID" IN ('E302906','G260465')) ;
+
+select  *    from cqc."WorkerQualification" where "TribalID" in (170258, 196840);
```
Comment (wozzer72) [inline Live_data_update.sql]: You'll be expecting zero on everything here, so returning a count makes more sense.

File: Live_data_update.sql (hunk 1 above)
Comment (Shakir-Sopra) [reply Live_data_update.sql]: Agree, I will edit it to make it count instead of returning the whole records.

Thanks
Shakir

Comment (Shakir-Sopra) [reply Live_data_update.sql]: Hi Warren 

I have changed the records to counting the records.

Thanks
Shakir

**Pull request merged**